packages/ember-glimmer/lib/ember-views/child-views-support.js

```diff
@@ -0,0 +1,73 @@
+/**
+@module ember
+@submodule ember-views
+*/
+import { Mixin } from 'ember-metal/mixin';
+import { get } from 'ember-metal/property_get';
+import { getOwner, setOwner, OWNER } from 'container/owner';
+
+export default Mixin.create({
+  init() {
+    this._super(...arguments);
+
+    /**
+      Array of child views. You should never edit this array directly.
+      Instead, use `appendChild` and `removeFromParent`.
+
+      @property childViews
+      @type Array
+      @default []
+      @private
+    */
+    this.childViews = [];
+    this.ownerView = this.ownerView || this;
```

Comment:
is it somewhat risky to allow this to fallback to `this`, an accidentally null/undefined `ownerView` will now just (likely with only subtle issues) fall back to `this`.